source/analysis/AM3DAdditionAB.cpp

```diff
@@ -253,5 +257,6 @@ void AM3DAdditionAB::computeCachedValues() const
             cachedData_[j] += data.at(j);
     }
 
+    cachedDataRange_ = AMUtility::rangeFinder(cachedData_);
```

Comment:
Probably.  I'd need to modify the caching to ensure a single invalid value is used as I believe it is not just -1 like the rest of them.  I probably need to do something to ensure that if the value is -1 then the data is always -1.